src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -261,12 +269,131 @@ private static File createInOrderSampleMap() {
         return getSampleMapFile(sampleFileContents);
     }
 
-    private static File getSampleMapFile(String sampleFileContents) {
+    private static File getSampleMapFile(final String sampleFileContents) {
         final File sampleNameMap = IOUtils.writeTempFile(sampleFileContents, "sampleNameMap", ".txt");
         sampleNameMap.deleteOnExit();
         return sampleNameMap;
     }
 
+    private static File getSampleMapFile(final Map<String, String> mapping){
+        return getSampleMapFile(mapping.entrySet()
+                .stream()
+                .map( pair -> pair.getKey() + "\t" + pair.getValue())
+                .collect(Collectors.joining("\n")));
+    }
+
+    @DataProvider
+    public static Iterator<Object[]> getRenameCombinations() {
+        final Map<String,String> noRemapping = new LinkedHashMap<>();
+        noRemapping.put("s1", "s1");
+        noRemapping.put("s2", "s2");
+        noRemapping.put("s3", "s3");
+
+        final Map<String,String> sameInput = new LinkedHashMap<>();
+        sameInput.put("s1", "s1");
+        sameInput.put("s2", "s1");
+        sameInput.put("s3", "s1");
+
+
+        final Map<String,String> sameInputWeirdOrder = new LinkedHashMap<>();
+        sameInputWeirdOrder.put("s3", "s1");
+        sameInputWeirdOrder.put("s1", "s1");
+        sameInputWeirdOrder.put("s2", "s1");
```

Comment:
What currently happens if there are duplicate sample names in the sample name map (ie., on the left side of the mapping)? Add a separate test case for this with (I would hope) an expected exception.

Reply:
We have a test for this in `GenomicsDBImportUnitTest.testBadInputFiles`